### PR TITLE
Added theming feature

### LINKED ISSUE #90: Als gebruiker van de archiefbeheercomponent wil ik dat deze getoond kan worden in de huisstijl van mijn gemeente,
zodat deze aansluit bij de uitstraling van het gemeentelijke applicatielandschap.

*Clarification*

There should be a style that is whitelabel, and stylable in any municipality theme. At minimum, remove Utrecht specifici style (use neutral colors) and allow to configure these colors and a logo.

### PR DESCRIPTION
**Pull request by joeribekker**
Fixes #90

Needs to be rebased once #127 is merged.

image

### CODE REVIEW

**Comment by SilviaAmAm** [comment]
I made a few comments in the commits. 

Some additional remarks:
- I think it may be good to add some documentation about how the frontend colours can now be configured.
- In the front page I see that after the buttons are clicked they change colour (the buttons are actually links). I find that this colour makes it a bit hard to read.
Screenshot from 2021-02-17 12-16-59

File: src/archiefvernietigingscomponent/theme/models.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "Image will be scaled down to a height of 100 pixels. Typically upload a picture with a height of 200 pixels."
+        ),
+    )
+
+    organization_name = models.CharField(max_length=100)
+    tagline = models.CharField(max_length=200, blank=True)
+    phone_number = models.CharField(max_length=50, blank=True)
+    address = models.CharField(max_length=400, blank=True)
+
+    color_primary = models.CharField(
```
[the remark this answers is not in the document]
Comment (joeribekker) [reply src/archiefvernietigingscomponent/theme/models.py:51]: Added help texts.

File: src/archiefvernietigingscomponent/theme/models.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+class ThemeConfig(SingletonModel):
+    logo = models.FileField(
+        upload_to="theme/",
+        help_text=_(
+            "Image will be scaled down to a height of 100 pixels. Typically upload a picture with a height of 200 pixels."
+        ),
+    )
+
+    organization_name = models.CharField(max_length=100)
```
[the remark this answers is not in the document]
Comment (joeribekker) [reply src/archiefvernietigingscomponent/theme/models.py]: Added verbose names.

**Comment by joeribekker** [comment]
> In the front page I see that after the buttons are clicked they change colour (the buttons are actually links). I find that this colour makes it a bit hard to read.

Was caused by the start page refactor to inherit from master. I made it on its own now since its completely separate from the app styling.

File: src/archiefvernietigingscomponent/theme/models.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        upload_to="theme/",
+        help_text=_(
+            "Image will be scaled down to a height of 100 pixels. Typically upload a picture with a height of 200 pixels."
+        ),
+    )
+
+    organization_name = models.CharField(
+        _("Organization name"),
+        max_length=100,
+        help_text=_("Shown on the top of the footer."),
```
Comment (SilviaAmAm) [inline src/archiefvernietigingscomponent/theme/models.py:28]: Maybe better (so it's also consistent with the address helptext):
```suggestion
        help_text=_("Shown at the top of the footer."),
```

**Review by SilviaAmAm — COMMENTED** [review]
I also noticed that now the "Afmelden" link in the demo pane doesn't work anymore (caused by the fact that the demo users are no longer staff). So this link should probably be removed for now and then maybe we should deal with it in another PR?

**Comment by joeribekker** [comment]
> I also noticed that now the "Afmelden" link in the demo pane doesn't work anymore (caused by the fact that the demo users are no longer staff). So this link should probably be removed for now and then maybe we should deal with it in another PR?

Yes, the above is not caused by this PR :)

**Review by SilviaAmAm — APPROVED** [review]

**Pull request merged**